steps/infra/aws/main.tf

```diff
@@ -41,6 +41,12 @@ module "masters" {
   user_data_igns      = "${var.tectonic_ignition_masters}"
 }
 
+module "iam" {
+  source = "../../../modules/aws/master"
```

Comment:
We're already importing this module up above:

```console
$ git grep -n aws/master origin/pr/265 -- steps/infra
origin/pr/265:steps/infra/aws/main.tf:19:  source = "../../../modules/aws/master"
origin/pr/265:steps/infra/aws/main.tf:45:  source = "../../../modules/aws/master"
```

Just set your new variable there instead of adding this `module` block here.

Reply:
Copy paste error. Fixed.